ViewModel/FrequencyRelationships/GraphicalVM.cs

```diff
@@ -0,0 +1,89 @@
+﻿using paireddata;
+using Statistics.GraphicalRelationships;
+using System.Xml.Linq;
+using Statistics.Distributions;
+using HEC.FDA.ViewModel.TableWithPlot.Rows;
+using System;
+using HEC.MVVMFramework.ViewModel.Implementations;
+using HEC.FDA.ViewModel.TableWithPlot;
+using HEC.FDA.ViewModel.TableWithPlot.Data;
+
+namespace HEC.FDA.ViewModel.FrequencyRelationships
+{
+    public class GraphicalVM : ComputeComponentVM
+    {
+        private int _equivalentRecordLength = 5;
+        private NamedAction _confidenceLimits;
+        public NamedAction ConfidenceLimits { get { return _confidenceLimits; } set { _confidenceLimits = value; NotifyPropertyChanged(); } }
+
+        public GraphicalUncertainPairedData MyGraphical
+        {
+            get{return new GraphicalUncertainPairedData(((GraphicalDataProvider)SelectedItem).Xs, ((GraphicalDataProvider)SelectedItem).Ys, EquivalentRecordLength,new CurveMetaData(), usingStagesNotFlows: true);}
+           
+        }
+        public int EquivalentRecordLength
+        {
+            get{ return _equivalentRecordLength;}
+            set
+            {
+                _equivalentRecordLength = value;
+                NotifyPropertyChanged();
+            }
+        }
+        public GraphicalVM(string name, string xlabel, string ylabel) : base(name)
+        {
+            Options.Clear();
+            Options.Add(new GraphicalDataProvider());
+            SelectedItem = Options[0];
+            Initialize();
+        }
+        public GraphicalVM(XElement vmEle)
+        {
+            LoadFromXML(vmEle);
+            Initialize();
+        }
+        private void Initialize()
+        {
+            ConfidenceLimits = new NamedAction();
+            ConfidenceLimits.Name = "Compute Confidence Limits";
+            ConfidenceLimits.Action = ConfidenceLimitsAction;
+        }
+        override public XElement ToXML()
+        {
+            XElement ele = base.ToXML();
+            ele.SetAttributeValue("EquivalentRecordLength", EquivalentRecordLength);
+            return ele;
+        }
+        override public void LoadFromXML(XElement element)
+        {
+            EquivalentRecordLength = int.Parse(element.Attribute("EquivalentRecordLength").Value);
+            base.LoadFromXML(element);
+        }
+        private void ConfidenceLimitsAction(object arg1, EventArgs arg2)
+        {
+            GraphicalUncertainPairedData graffical = MyGraphical;
+            PairedData upperNonExceedence = graffical.SamplePairedData(.95) as PairedData;
+            PairedData lowerNonExceedence = graffical.SamplePairedData(.05) as PairedData;
```

Comment:
I think for consistency purposes we should use the .025 and .975

Reply:
This has been corrected.